application/use_cases/stock_locations/create_stock_location/create_stock_location_interactor.py

```diff
@@ -0,0 +1,23 @@
+from clapy import Interactor
+
+from application.dtos.stock_location_dto import get_stock_location_dto
+from application.services.ipersistence_context import IPersistenceContext
+from domain.entities.stock_location import StockLocation
+
+from .create_stock_location_input_port import CreateStockLocationInputPort
+from .icreate_stock_location_output_port import ICreateStockLocationOutputPort
+
+
+class CreateStockLocationInteractor(Interactor):
+
+    def __init__(self, persistence_context: IPersistenceContext):
+        self.persistence_context = persistence_context
+
+    async def execute_async(self, input_port: CreateStockLocationInputPort, output_port: ICreateStockLocationOutputPort):
+        stock_location = StockLocation(
```

Comment:
same style guide as work for local variables

Reply:
should be good now